test/runUnitTests.js

```diff
@@ -16,4 +16,10 @@
 
 import {unitTestRunner} from './unit/unitTestRunner.js';
 
-unitTestRunner.parseOptionsAndRun();
\ No newline at end of file
+unitTestRunner.run().catch((ex) => {
+  if (typeof ex === 'number') {
```

Comment:
`catch()` will be invoked by the 'error' handler.  I fixed the number of failures and tested both paths.